### ISSUE TITLE
Elevated plasma soluble FAS ligand level

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10229]
**Preferred term label:**

**Synonyms**

Elevated level of Plasma soluble FASL
Increased soluble FASL

**Definition (free text, please give PubMed ID)**
PMID: 30958694 Elevated plasma-soluble FAS ligand levels (.200 pg/mL)

PMID: 20301287 Secondary accessory criteria:  Elevated levels of one of the following: **Plasma soluble FASL**

**Parent term (use hpo.jax.org/app)**
Abnormal cytokine signaling HP:0031406

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Autoimmune lymphoproliferative syndrome (ALPS) ORPHA:3261

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000017
- label: Elevated circulating soluble FASL concentration
- parents: HP:0010876
- pmid: PMID:30958694|PMID:20301287
- def: The concentration of soluble FASL in the blood circulation is above the upper limit of normal. The first apoptosis signal receptor (FAS) pathway regulates apoptosis and is critical for proper development and functioning of the immune system. FAS ligand (FASL) can bind to FAS receptors.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10229
- synonyms:
